openshift/release/generate-release.sh

```diff
@@ -5,6 +5,8 @@ set -euo pipefail
 source $(dirname $0)/resolve.sh
 
 GITHUB_ACTIONS=true $(dirname $0)/../../hack/update-codegen.sh
+git apply openshift/patches/disable-ko-publish-rekt.patch
```

Comment:
This is ok to not be applied, we can remove it

Reply:
We're using the images file flag and so ko will not be used

Reply:
ref for main https://github.com/openshift-knative/eventing/pull/605

Reply:
Ok. thanks - I guess it should be consistent than. I will update.